source/common/upstream/cluster_manager_impl.cc

```diff
@@ -536,6 +538,11 @@ ClusterManagerImpl::ThreadLocalClusterManagerImpl::ClusterEntry::ClusterEntry(
                                        parent.parent_.random_));
     break;
   }
+  case LoadBalancerType::OriginalDst: {
```

Comment:
Can you make sure this code has coverage? (Depending on how you wrote tests it might not).